compiler/il/OMRNode.cpp

```diff
@@ -8642,3 +8694,43 @@ OMR::Node::resetFlagsForCodeMotion()
 /**
  * Node flags functions end
  */
+
+bool
+OMR::Node::isLoadOfStaticFinalField()
+   {
+   if (self()->getOpCode().isLoadDirect() &&
```

Comment:
You may want to check that a static field is loaded via a direct load on all platforms, in particular on System Z. I believe we have an api to load statics indirectly as well (exposing the address as the child of the indirect load on platforms where we want to common the address).

Reply:
Thanks for pointing this out, we do use indirect load for a static field on Z9. I will remove this check. Just for a record, the tree format of an indirect load of static field is
```
xloadi <static field symref>
   loadaddr <class statics>
```